ml-prims/test/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ add_executable(mlcommon_test
   transpose.cu
   unary_op.cu
   weighted_mean.cu
+  host_buffer.cu
```

Comment:
minor knitpick! please keep the files in alphabetical order.

Reply:
Will fix it.